packages/clipboard/src/index.ts

```diff
@@ -12,31 +27,51 @@ import * as clipboardpoly from 'clipboard-polyfill';
  */
 export class Clipboard {
   /**
-   * This copies the given data to the system clipboard and as fallback,
-   * it copies to local storage.
-   * @param data data that needs to be copied to the clipboard
+   * CliboardType System - This copies the given data to the system clipboard.
+   * ClipboardType Local - This copies the given data to the local storage.
+   * ClipboardType Both - This copies the given data to the system clipboard
+   * and as fallback, it copies to local storage.
+   * @param data data that needs to be copied to the clipboard.
+   * @param clipboardtype type of clipboard to be used.
    */
-  public copy(data: any): Promise<any> {
-    return clipboardpoly.readText()
-      .then(() =>  clipboardpoly.writeText(data))
-      .catch(() => this.storeToLocalClipboard(data));
+  public copy(data: any, clipboardtype: ClipboardType = ClipboardType.System) {
+    if (clipboardtype === ClipboardType.System || clipboardtype === ClipboardType.Both) {
+      clipboardpoly.writeText(data).catch(() => {
+        if (clipboardtype === ClipboardType.Both) {
+          this.storeToLocalClipboard(data);
```

Comment:
If clipboard type is system, why are we copying it to the local clipboard? We need to organize the code a little bit to avoid these issues.

Reply:
I have put a condition to check if it is both only it should store it in local.